acceptance-tests/features/support/functions.rb

```diff
@@ -28,3 +28,61 @@ def delete_deed_data(deed_id)
             "received response #{response.code}."
   end
 end
+
+def sign_the_deed(deed_id, signature, borrower_id)
+  signature_json = {
+    'signature' => signature
+  }
+  response = HTTP.post(Env.domain + '/deed/' + deed_id.to_s +
+                       '/' + borrower_id + '/signature/',
+                       json: signature_json)
+
+  if response.code == 200
+    puts "Deed #{deed_id} has been signed."
+  else
+    fail "Error: Couldn't sign deed #{deed_id}, "\
+            "received response #{response.code}."
+  end
+end
+
+def make_deed_effective(deed_id)
+  signature_json = {
+    'registrars-signature' => 'SIGNATURE'
+  }
+  response = HTTP.post(Env.domain + '/deed/' + deed_id.to_s + '/completion',
+                       json: signature_json)
+
+  if response.code == 200
+    puts "Deed #{deed_id} has been made effective."
+  else
+    fail "Error: Couldn't make deed #{deed_id} effective, "\
+            "received response #{response.code}."
+  end
+end
+
+def create_case_data
+  payload = {
+    'conveyancer_id' => '1'
+  }
+  response = HTTP.post(Env.case_api + '/case', json: payload)
+  if response.code == 201
+    JSON.parse(response.body)['id']
+  else
+    fail "Error: Couldn't create case #{payload}, "\
+            "Received response #{response.code}"
+  end
+end
+
+def update_case_deed
+  payload = {
+    'deed_id' => @deed_id
+  }
+  response = HTTP.post(Env.case_api + '/case/' + @case_id.to_s + '/deed',
+                       json: payload)
+  if response.code == 200
+    JSON.parse(response.body)['id']
+  else
+    fail "Error: Couldn't create case #{payload}, "\
+            "Received response #{response.code}"
```

Comment:
I think that this fail message has been carried over from create_case function. Should be reworded to something more relevant to linking the deed and case such as "Error: Couldn't link deed #{@deed_id} to case #{case_id}......." second line is fine though.